README.md

```diff
@@ -1 +1,37 @@
-# capstone-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Capstone Project
+Desktop and Mobile web based on https://www.behance.net/gallery/29845175/CC-Global-Summit-2015
+
```

Comment:
- [ ] I would recommend giving here the description of your project.

- [ ] Also, The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

- [ ] A link to the deployed project should be present in the README. It can be deployed on the application using Heroku, Netlify, or GitHub Pages.